scripts/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2022 MosaicML LLM Foundry authors
```

Comment:
why do we need to make scripts a module? Would be better to not, and anything that needs to be importable move into the llmfoundry folder. If its just needed for tests, you can just append to the path to make things importable.